datumaro/plugins/data_formats/imagenet.py

```diff
@@ -56,11 +56,55 @@ def _load_items(self, path):
 
 
 class ImagenetImporter(Importer):
+    """TorchVision's ImageFolder style importer.
+    For example, it imports the following directory structure.
+
+    root
+    ├── label_0
+    │   ├── label_0_1.jpg
+    │   └── label_0_2.jpg
+    └── label_1
+        └── label_1_1.jpg
+    """
+
     @classmethod
     def find_sources(cls, path):
         if not osp.isdir(path):
             return []
-        return [{"url": path, "format": ImagenetBase.NAME}]
+
+        for _ in find_images(path, recursive=True, max_depth=1):
```

Comment:
how it will be operated when setting `max_depth=2`? If so, can we be able to support subset directories?

Reply:
No, it may be like the following example.

```
root
 - label_0
     - abc
        - 0.jpg
        - 1.jpg
     - def
        - 0.jpg
  - label_1
  ...
```

```python
Dataset = {
"subset": default,
"items": [ DatasetItem(media=Image("label_0/abc/0.jpg", annotation=Label("label_0"), ... ]
}
```